crud.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_author(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author_by_name(db: Session, name):
+    return db.query(models.DBAuthor).filter(models.DBAuthor.name == name).first()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio,
+    )
+
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_book_list(
+    db: Session, skip: int = 0, limit: int = 10, author: Optional[str] = None
```

Comment:
The `author` parameter in the `get_book_list` function is intended to filter books by author name. Ensure that the `models.DBBook.author` field is a string that matches the `author` parameter. If `models.DBBook.author` is a foreign key or another type, adjust the filter logic accordingly.